.github/CODEOWNERS

```diff
@@ -54,6 +54,9 @@ gradlew.bat @BenHenning
 /app/src/main/res/**/*.xml @rt4914
 /utility/src/main/res/**/*.xml @rt4914
 
+# App UI strings.
+/app/src/main/res/values*/strings.xml @BenHenning
```

Comment:
Btw, to your question -- I think it is probably fine. This doc says glob patterns here follow gitignore syntax, with some exceptions: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax -- and this doc lists the exceptions: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#syntax-exceptions.

If it doesn't work out, perhaps having an additional subdirectory like /app/src/main/res/values/ar might be a reasonable alternative?

Reply:
Having each subdirectory is a bit annoying, but it'll work as a backup if we remove the glob xml patterns (which is basically just a bandaid over the current situation of trying to wrangle the large number of res directories).

I came to the same conclusion as you re: compatibility, but I vaguely remember trying something like this in the past & it not working. We'll see if it does this time.